README.md

```diff
@@ -13,43 +16,196 @@ This project is still heavily under development and is not ready for use
 **yet**. However, if you want to experiment, please do try running the current
 development build and reporting any issues you run into.
 
-## Requirements
+## Quickstart
+
+> Prebuilt images for cert-manager are made available on Dockerhub.
+
+This guide sets up cert-manager to run as a Deployment on your Kubernetes
+cluster. It will then go on to set up the Letsencrypt staging server as a
+Certificate issuer, and request a Certificate for a domain you control using
+the HTTP01 challenge mechanism.
+
+This guide will configure cert-manager to attempt to fulfill Certificate
+resources in all namespace in your cluster.
+
+### 0. Pre-requisites
 
 * Kubernetes cluster with CustomResourceDefinitions enabled (1.7+) (see [#49](https://github.com/jetstack-experimental/cert-manager/issues/49))
 
-## Deploying
+### 1. Deploy cert-manager
 
-To deploy the latest development version, run:
+To deploy the latest version of cert-manager, run:
 
 ```
-$ kubectl create -f docs/cert-manager.yaml
+$ kubectl create -f https://github.com/jetstack-experimental/cert-manager/blob/master/docs/cert-manager.yaml
 ```
 
-## Getting started
+In future this may be replaced with a Helm chart. There are currently no
+official RBAC roles defined for cert-manager (see [#34](https://github.com/jetstack-experimental/cert-manager/issues/34))
+
+### 2. Set up letsencrypt staging issuer
+
+An Issuer in cert-manager describes a source for signed TLS certificates that
+cert-manager can use to fulfil Certificate resources in a Kubernetes cluster.
+
+Within the Issuers spec, we can define any configuration that may be required
+(eg. credentials for updating a DNS server) on a per-issuer basis.
+
+In the below example, you **must** remember to fill in the `spec.acme.email`
+field.
+
+```yaml
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: letsencrypt-staging
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-staging.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: ""
+    # Name of a secret used to store the ACME account private key
+    privateKey: letsncrypt-staging
+    # ACME dns-01 provider configurations
+    dns-01:
+      # Here we define a list of DNS-01 providers that can solve DNS challenges
```

Comment:
❓ But in the intro we say that we'll be using the HTTP01 challenge mechanism. This looks like a DNS challenge mechanism. Or am I misunderstanding?